### ISSUE TITLE
Accessibility Audit: Filter checkboxes don't show up in Contrast Mode

### ISSUE THREAD

**Issue by jnowell** [CityOfDetroit/bloom#929]
In a Macbook, turn on Increase Contrast (System Preferences -> Accessibility -> Display -> 'Increase Contrast Mode'). Now go to Firefox and look at the Filter modal. The checkboxes that are included in the modal (for Bedrooms, Regions, Accessibility Features, and 'Show Homes With Missing Information') don't show up. 

NOTE: I was not able to replicate this with Chrome. 

<img width="712" alt="Screen Shot 2021-12-29 at 4 46 28 PM" src="https://user-images.githubusercontent.com/394908/147705999-962a255f-a7b5-4c51-a1d7-401c3df1458c.png">
